### DISCUSSION TITLE
passing 'zfs set' arguments during replication.  how to do it.

### DISCUSSION THREAD

**Discussion by hput** [psy0rz/zfs_autobackup#134] [Q&A]
Using the example in the wiki for sending zfs-autobackup out on a replication mission:

   zfs-autobackup -v --ssh-source pve01 offsite1 data/backup/pve01

Is it possible to `zfs set canmount=noauto' on each zfs data set as it is pulled?

Or some other smarter way to get that setting into play.  Maybe "zfs set canmount=noauto  data/backup/pve01" prior to running the replication pull?.  Would that have the effect of forcing all datasets pulled in to inherit that canmount setting.

Or some other way to keep the incoming data sets from blocking the backup servers' own OS (zfs on root) from there mountpoints.
None of this seems to be taken up in any of the wiki examples.... Only in tips.  And not really explained there.  At least not so a common dumb-bell can understand it.

**Comment by Scrin** [comment]
According to the [usage](https://github.com/psy0rz/zfs_autobackup/wiki#usage), you can use `--set-properties` to set properties during replication
